Polyhedron/demo/Polyhedron/Plugins/Surface_mesh_deformation/Scene_edit_polyhedron_item.cpp

```diff
@@ -245,6 +245,9 @@ void Scene_edit_polyhedron_item_priv::initializeBuffers(CGAL::Three::Viewer_inte
         item->buffers[Facet_normals].release();
         item->vaos[Facets]->release();
         program->release();
+
+        if(offset.norm() !=0)
+          delete vertices;
```

Comment:
1. I do not understand why this PR about the loading of c3t3 items modifies this file in cabf3fb0dbb98f8671c620779c81994de31070d3.
2. Those lines are compilation error:
```
test/Polyhedron_Demo/Plugins/Surface_mesh_deformation/Scene_edit_polyhedron_item.cpp:250:11: 
error: cannot delete expression of type 'std::vector<GLdouble>'
          delete vertices;
          ^      ~~~~~~~~
1 error generated.
```
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.10-Ic-122/Polyhedron_Demo/TestReport_cgaltester_x86-64_Darwin-13.0_Apple-clang-5.0_Release.gz